packages/remix/package.json

```diff
@@ -15,19 +15,34 @@
   ],
   "author": "Clerk",
   "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "exports": {
+    ".": "./dist/index.js",
+    "./ssr.server": "./dist/ssr/index.js",
+    "./api.server": "./dist/api/index.js"
```

Comment:
Inside Remix v2 you'd have to import those with the `.js` file extension since no exports map was setup. This fixes it.

Remix complained with `None of the conditions provided ("types", "require") match any of the currently active conditions ("browser", "default", "import", "module")` when using this:

```json
{
  ".": {
     "types": "bla",
     "require": "bla"
   }
}
```

So we use `typesVersions instead